tests/11register.pl

```diff
@@ -227,7 +227,7 @@ sub add_email_for_user {
             three_pid_creds => {
                id_server       => $id_server->name,
                id_access_token => $id_access_token,
-               sid             => $sid,
+               sid             => "$sid",
```

Comment:
this isn't the only place that `sid` is passed from `validate_email` to `do_request_json` - if we're going to change it, we should probably change all of them. However, I'm surprised it's necessary - isn't it already returned as a string by `validate_email` (provided `/r0/account/3pid/email/requestToken` returns it as a string)?

Anyway, this feels like an orthogonal change to the configuration. Please could you make it a separate PR?

Reply:
I will check again `/r0/account/3pid/email/requestToken`. It could be my implementation that is bad.